src/main/java/banking/configuration/DBManager.java

```diff
@@ -0,0 +1,48 @@
+package banking.configuration;
+
+import org.sqlite.SQLiteDataSource;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class DBManager {
+
+    private static final String CREATE_TABLE = """
+            CREATE TABLE IF NOT EXISTS account
+            (id INTEGER PRIMARY KEY, 
+            number TEXT NOT NULL, 
+            pin TEXT NOT NULL, 
+            balance INTEGER DEFAULT 0)""";
+
+    private String dbName;
+    private String url;
+    private final SQLiteDataSource dataSource;
+
+    public DBManager(String dbName) {
+        dataSource = new SQLiteDataSource();
+        this.dbName = dbName;
+        createDatabase();
+        createTable();
+    }
+
+    public void createDatabase() {
```

Comment:
Why this method has public access modifier?

Reply:
fixed